spring-integration-file/src/main/java/org/springframework/integration/file/FileWritingMessageHandler.java

```diff
@@ -101,6 +105,9 @@
 public class FileWritingMessageHandler extends AbstractReplyProducingMessageHandler
 		implements Lifecycle, MessageTriggerAction {
 
+	private static final boolean nioFilesPresent = ClassUtils.isPresent("java.nio.file.Files",
+			FileWritingMessageHandler.class.getClassLoader());
```

Comment:
Looks like an artifact of the fact I rebased this branch to master and pushed before I pushed your PR to master.

I would have thought GH was smart enough to figure it out.